rust/toolchain.bzl

```diff
@@ -35,7 +41,19 @@ def _rust_stdlib_filegroup_impl(ctx):
         #
         # alloc depends on the allocator_library if it's configured, but we
         # do that later.
-        dot_a_files = [_make_dota(ctx, f) for f in std_rlibs]
+        #
```

Comment:
Small nit since this seems separate from the block comment (but might be just me)
```suggestion

```

Reply:
Thank you! I'll look into moving the `panic_unwind` vs `panic_abort` logic out of `rust_stdlib_filegroup` and report back.

Reply:
Moved the panic filtering logic in `_make_libstd_and_allocator_ccinfo`. I think I'm satisfied with this approach until we have a more flexible way to let the user specify the panic strategy.